src-cpp/app/modules/shared/section_item.h

```diff
@@ -0,0 +1,118 @@
+#ifndef SECTION_ITEM_H
+#define SECTION_ITEM_H
+
+#include <QObject>
+#include <QString>
+
+namespace Shared
+{
+namespace Models
+{
+enum SectionType
+{
+    Chat = 0,
+    Community,
+    Wallet,
+    Browser,
+    ProfileSettings,
+    NodeManagement
+};
+class SectionItem: public QObject
+{
+    Q_OBJECT
```

Comment:
This can be a `Q_GADGET`?

Reply:
can use it cause I have NOTIFY for one of the Q_PROPERTY?